src/surface_conditions/surface_conditions.jl

```diff
@@ -21,50 +21,151 @@ function update_surface_conditions!(Y, p, t)
     )
     int_local_geometry_values =
         Fields.field_values(Fields.level(Fields.local_geometry_field(Y.c), 1))
-
     (; ᶜts, ᶜu, sfc_conditions, sfc_setup, params, atmos) = p
     int_ts_values = Fields.field_values(Fields.level(ᶜts, 1))
     int_u_values = Fields.field_values(Fields.level(ᶜu, 1))
     int_z_values =
         Fields.field_values(Fields.level(Fields.coordinate_field(Y.c).z, 1))
     sfc_conditions_values = Fields.field_values(sfc_conditions)
 
+    is_prognostic_sfc = atmos.surface_model isa PrognosticSurfaceTemperature
+    possible_sfc_prognostic_temp =
```

Comment:
Any reason why you have a `possible` in the name?

Reply:
It's meant to explain sfc_prognostic_temp is nothing in many cases, although I'm not sure it's necessary. What do you think?

Reply:
I think just something like `prognostic_sfc_temp` would be good.